docs/specifications/update.md

```diff
@@ -0,0 +1,191 @@
+# Update Command
+
+`ng update` is a new command in the CLI to update one or multiple packages, its peer dependencies, and the peer dependencies that depends on it.
+
+If there are inconsistencies, for example if peer dependencies cannot be matches by a simple semver range, the tool will error out (and nothing will be committed on the filesystem).
+
+## Glossary
+
+* **migration**. A schematic that is performed on the 
```

Comment:
Sentence fragment.
If there are no other terms for the glossary consider omitting it.

Reply:
Removed the whole Glossary.